### PR TITLE
Copy external content into the docs repository

### PR DESCRIPTION
**Pull request by marians**
With the aggregated changelog we decided to copy content into the docs repository.

Having the content inside one source tree, updateable through automation, would also help achieving https://github.com/giantswarm/giantswarm/issues/14401 (automatic last modified date for articles), which requires every file to be inside the source tree.

This PR also removes the `make build` target, since there is no longer a need to pull different sources into a temp dir at build time. Building happens based on the src tree now.

Depends on https://github.com/giantswarm/docs-indexer/pull/32

### CODE REVIEW

**Review by axbarsan — APPROVED** [review]

File: src/layouts/shortcodes/gsctl_version.html [hunk 1]
```diff
@@ -1 +1 @@
-GSCTL_VERSION
\ No newline at end of file
+0.26.0
```
Comment (marians) [inline src/layouts/shortcodes/gsctl_version.html:1]: FYI: This file is generated. The trailing newline missing is on purpose.

File: src/content/reference/web-interface/tenant-cluster-labelling.md [hunk 2]
```diff
@@ -5,9 +5,6 @@ date: 2020-11-18
 last_review_date: 2020-05-29
 layout: subsection
 weight: 20
-aliases: [
-    "/reference/web-interface/tenant-cluster-labelling/",
-]
```
Comment (marians) [inline src/content/reference/web-interface/tenant-cluster-labelling.md:10]: Removed, as it does not make sense. Caused a warning like "two pages want to write to the same path".

File: scripts/update-crd-reference/config.yaml [hunk 3]
```diff
@@ -3,7 +3,7 @@ source_repository:
   url: https://github.com/giantswarm/apiextensions
   organization: giantswarm
   short_name: apiextensions
-  commit_reference: v3.2.0
+  commit_reference: v3.10.0
```
Comment (marians) [inline scripts/update-crd-reference/config.yaml:6]: Setting to the latest version while at it

File: README.md [hunk 4]
```diff
@@ -8,13 +8,19 @@ This repository named `docs` holds the main **content** of our documentation. Th
 site is created using the static site generator [HUGO](http://gohugo.io/) based on Markdown files
 in the `src/content/` directory. Images are in the `src/static/img/` directory.
 
-Additional content repositories are tied in through the file `src/external-repositories.txt`.
+Additional content is tied in through the scripts
 
-The `src/` directory also contains HTML templates and the HUGO site configuration.
+- `scripts/aggregate-changelogs`: Aggregates changelog entries into the `src/content/changes` destination.
+- `scripts/update-crd-reference`: Generates reference pages for our custom resource definitions in the `src/content/reference/cp-k8s-api` destiantion.
```
Comment (axbarsan) [inline README.md:14]:
```suggestion
- `scripts/update-crd-reference`: Generates reference pages for our custom resource definitions in the `src/content/reference/cp-k8s-api` destination.
```

**Review by axbarsan — APPROVED** [review]

**Comment by marians** [comment]
Thanks for reviewing @axbarsan !

@giantswarm/chapter-front-end In order to complete this, https://github.com/giantswarm/docs-indexer/pull/32 also needs a review. Thanks in advance!

**Pull request merged**